package.json

```diff
@@ -56,7 +57,8 @@
     "tailwindcss": "^3.3.6",
     "ts-node": "^10.9.2",
     "typescript": "5.7.3",
-    "vite": "^6.2.0"
+    "vite": "^6.2.0",
+    "vitest": "^3.0.8"
```

Comment:
All new packages to start with v from now on

Reply:
@jamiehenson switching to _**V**im_?